packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -175,7 +197,7 @@ export async function getDynamicEventType(input: TGetScheduleInputSchema) {
 }
 
 export function getRegularOrDynamicEventType(input: TGetScheduleInputSchema) {
-  const isDynamicBooking = !input.eventTypeId;
+  const isDynamicBooking = input.usernameList && input.usernameList.length > 1;
```

Comment:
In the frontend (but SSR) we decide if it's dynamic using different sentence. I think we should use the same.
![image](https://github.com/calcom/cal.com/assets/4461358/19e3f8d3-400b-4886-b667-aa4e32c89430)